src/viur/core/render/json/default.py

```diff
@@ -47,7 +47,7 @@ def render_structure(structure: dict):
                     "boundslng": "boundsLng",
                     "emptyvalue": "emptyValue",
                     "max": "maxAmount",
-                    "maxlength": "maxLength",
+                    "maxlength": "max_length",
```

Comment:
This is invalid, as this is the compatibility for the old bone structure rendering, and there, "maxLength" is the old value required.
```suggestion
                    "maxlength": "maxLength",
```